custom_components/teslafi/sensor.py

```diff
@@ -246,6 +246,26 @@
         available=lambda u, d, h: u and d.tpms.rear_right,
     ),
     # endregion
+    # region TeslaFi API Counts
+        TeslaFiSensorEntityDescription(
```

Comment:
If you want these to be seen as numeric, I think you'll need to set a unit of measurement ("#" for example; or "wakes" and "commands") and a [state class](https://developers.home-assistant.io/blog/2021/09/20/state_class_total?_highlight=total#state-classes). Given the way these counts work, I think we should use state class `total_increasing`.

Otherwise I believe HA will treat these as discrete string values, so e.g. a history-graph card won't show a graph, but instead will show a bar where every distinct value is a different color. I think it's more useful as a numeric entity that can be shown on a graph